src/deploy/osp_deployer/settings/config.py

```diff
@@ -1,528 +1,561 @@
+        self.foreman_configuration_scripts = self.cloud_repo_dir + "/src"
+
+        self.sah_kickstart = self.cloud_repo_dir + "/src/mgmt/osp-sah.ks"
+        self.director_deploy_sh = self.foreman_configuration_scripts +\
+            '/mgmt/deploy-director-vm.sh'
+        self.dashboard_deploy_py = self.foreman_configuration_scripts +\
+            '/mgmt/deploy-dashboard-vm.py'
+
+        self.undercloud_conf = self.foreman_configuration_scripts +\
+            '/pilot/undercloud.conf'
+        self.install_director_sh = self.foreman_configuration_scripts +\
+            '/pilot/install-director.sh'
+        self.deploy_overcloud_sh = self.foreman_configuration_scripts + \
+            '/pilot/deploy-overcloud.py'
+        self.assign_role_py = self.foreman_configuration_scripts +\
+            '/pilot/assign_role.py'
+        self.network_env_yaml = self.foreman_configuration_scripts + \
+            '/pilot/templates/network-environment.yaml'
+        self.dell_storage_yaml = self.foreman_configuration_scripts + \
+            '/pilot/templates/dell-cinder-backends.yaml'
+        self.dell_env_yaml = self.foreman_configuration_scripts + \
+            '/pilot/templates/dell-environment.yaml'
+        self.neutron_ovs_dpdk_yaml = self.foreman_configuration_scripts + \
+            '/pilot/templates/neutron-ovs-dpdk.yaml'
+        self.static_ips_yaml = self.foreman_configuration_scripts + \
+            '/pilot/templates/static-ip-environment.yaml'
+        self.static_vip_yaml = self.foreman_configuration_scripts + \
+            '/pilot/templates/static-vip-environment.yaml'
+        self.sanity_ini = self.foreman_configuration_scripts + \
+            '/pilot/deployment-validation/sanity.ini'
+        self.ipxe_rpm = self.foreman_configuration_scripts + \
+            '/pilot/ipxe/ipxe-bootimgs-20151005-1.git6847232.el7.' \
+            'test.noarch.rpm'
+
+        # The NIC configurations settings are validated after the Settings
+        # class has been instanciated.  Guard against the case where the two
+        # fixed are missing here to prevent an exception before validation
+        nics_settings = self.get_nics_settings()
+        if 'nic_env_file' in nics_settings:
+            self.nic_env_file = nics_settings['nic_env_file']
+            self.nic_env_file_path = self.foreman_configuration_scripts + \
+                '/pilot/templates/nic-configs/' + self.nic_env_file
+        if 'sah_bond_opts' in nics_settings:
+            self.sah_bond_opts = nics_settings['sah_bond_opts']
+
+        # This particular section has been moved right after the nics_settings
+        # section in order to catch the mode used by the nic-environment file
+        dellnfv_settings = self.get_settings_section(
+            "Dell NFV Settings")
+        self.ovs_dpdk_enable = dellnfv_settings[
+            'ovs_dpdk_enable']
+        self.ovs_dpdk_role_list = dellnfv_settings[
+            'ovs_dpdk_role_list']
```

Comment:
I see this here & in both the  ini and the json files, but dont see it being used anywhere ? If so could we remove all references to it ?

Reply:
the same is true for both ovs_dpdk_role_list and ovs_dpdk_policy

Reply:
Agree. It was there for future implementation. Removed.